transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClient.kt

```diff
@@ -191,4 +191,12 @@ interface MessagingClient {
      * latest available history.
      */
     fun invalidateConversationCache()
+
+    /**
+     * Notify the Agent that the User is typing a message.
+     *
+     * @throws IllegalStateException if called before session was connected.
+     */
+    @Throws(IllegalStateException::class)
+    fun userIsTyping()
```

Comment:
Given my above comment, does it make sense to change this method signature? Maybe one of the following?

`fun indicateTyping()`

or

`fun sendTypingIndicator()`

Reply:
I will change to `sendTypingIndicator`

Reply:
I think I have an ever so slight preference for `indicateTyping()`.